docs/design/2025-05-26-refresh-stats-command.md

```diff
@@ -0,0 +1,322 @@
+# Refresh Stats Command
+
+- Author(s): [Poe Liu](http://github.com/0xPoe)
+- Discussion PR: https://github.com/pingcap/tidb/pull/61330
+- Tracking Issue: https://github.com/pingcap/tidb/issues/61273
+
+## Table of Contents
+
+- [Refresh Stats Command](#refresh-stats-command)
+  - [Table of Contents](#table-of-contents)
+  - [Introduction](#introduction)
+  - [Motivation or Background](#motivation-or-background)
+  - [Detailed Design](#detailed-design)
+    - [Guide-level explanation](#guide-level-explanation)
+    - [Reference-level explanation](#reference-level-explanation)
+      - [Goals](#goals)
+      - [Solution](#solution)
+        - [Broadcast Query Executor](#broadcast-query-executor)
+        - [Workflow](#workflow)
+        - [Concurrency Control](#concurrency-control)
+        - [Timeout](#timeout)
+      - [FAQ](#faq)
+  - [Test Design](#test-design)
+    - [Functional Tests](#functional-tests)
+    - [Scenario Tests](#scenario-tests)
+    - [Compatibility Tests](#compatibility-tests)
+    - [Performance Tests](#performance-tests)
+  - [Impacts \& Risks](#impacts--risks)
+  - [Investigation \& Alternatives](#investigation--alternatives)
+  - [Unresolved Questions](#unresolved-questions)
+
+## Introduction
+
+This document proposes a new SQL command to refresh the statistics for the specified tables. This command is useful for the BR tool to refresh the statistics for the tables that have been restored. It also can be used to debug in-memory statistics issues.
+
+## Motivation or Background
+
+Recently, the BR team has been working on a new solution to speed up statistics backup and restore for some key customers.
```

Comment:
There is no 'BR team' in [our community](https://github.com/pingcap/community/tree/master/teams), IMHO it's better to say something like 'we have another project that...'

Reply:
Good catch! How do you think about now?